apiserver/facades/client/client/status.go

```diff
@@ -527,8 +550,8 @@ func fetchAllApplicationsAndUnits(
 	return appMap, unitMap, latestCharms, nil
 }
 
-// fetchRemoteApplications returns a map from application name to remote application.
-func fetchRemoteApplications(st Backend) (map[string]*state.RemoteApplication, error) {
+// fetchConsumerRemoteApplications returns a map from application name to remote application.
+func fetchConsumerRemoteApplications(st Backend) (map[string]*state.RemoteApplication, error) {
```

Comment:
Actually, based on this definition I don't know why it's called that - isn't it all remote applications?